src/dex/balancer-v2/balancer-v2.ts

```diff
@@ -62,12 +62,32 @@ const fetchAllPools = `query ($count: Int) {
     first: $count
     orderBy: totalLiquidity
     orderDirection: desc
-    where: {totalLiquidity_gt: ${MIN_USD_LIQUIDITY_TO_FETCH.toString()}, totalShares_not_in: ["0", "0.000000000001"], id_not_in: ["0xbd482ffb3e6e50dc1c437557c3bea2b68f3683ee0000000000000000000003c6"], swapEnabled: true, poolType_in: ["MetaStable", "Stable", "Weighted", "LiquidityBootstrapping", "Investment", "StablePhantom", "AaveLinear", "ERC4626Linear", "Linear", "ComposableStable"]}
+    where: {
+      totalLiquidity_gt: ${MIN_USD_LIQUIDITY_TO_FETCH.toString()},
+      totalShares_not_in: ["0", "0.000000000001"],
+      id_not_in: [
+        "0xbd482ffb3e6e50dc1c437557c3bea2b68f3683ee0000000000000000000003c6"
+      ],
+       address_not_in: [
+        "0x0afbd58beca09545e4fb67772faf3858e610bcd0",
+        "0x2ff1a9dbdacd55297452cfd8a4d94724bc22a5f7",
+        "0xbc0f2372008005471874e426e86ccfae7b4de79d",
+        "0xdba274b4d04097b90a72b62467d828cefd708037",
+        "0xf22ff21e17157340575158ad7394e068048dd98b",
+        "0xf71d0774b214c4cf51e33eb3d30ef98132e4dbaa",
+      ],
+      swapEnabled: true,
+      poolType_in: [
+        "MetaStable", "Stable", "Weighted", "LiquidityBootstrapping", "Investment", "StablePhantom", "AaveLinear",
+        "ERC4626Linear", "Linear", "ComposableStable", "BeefyLinear", "GearboxLinear", "MidasLinear",
+        "ReaperLinear", "SiloLinear", "TetuLinear", "YearnLinear"
+      ]
+    }
   ) {
     id
     address
     poolType
-    tokens {
+    tokens (orderBy: index) {
```

Comment:
@mwamedacen , look. There was another quick solution for the problem we had with indexes :smiley: